src/Contracts/Repositories/ModelRepositoryInterface.php

```diff
@@ -78,6 +78,15 @@ public function firstWhereOrFail($column, $operator = null, $value = null, strin
      */
     public function where($column, $operator = null, $value = null, string $boolean = 'and'): Collection;
 
+    /**
+     * @param string  $column
+     * @param mixed[] $values
+     * @param string  $boolean
+     * @param bool    $not
+     * @return Collection<int, T>
+     */
+    public function whereIn(string $column, array $values, string $boolean = 'and', bool $not = false): Collection;
```

Comment:
* I don't think we need the `$boolean` argument since the repository doesn't do building, but only direct execution.
* I'm personally not a fan of the boolean `$not` switch. It's probably better to have a specific `whereNotIn` method.

Reply:
Totally agree. Will make some changes!